snyk-monitor/templates/deployment.yaml

```diff
@@ -70,8 +70,14 @@ spec:
               secretKeyRef:
                 name: {{ .Values.monitorSecrets }}
                 key: integrationId
-          - name: SNYK_NAMESPACE
+          - name: SNYK_WATCH_NAMESPACE
             value: {{ include "snyk-monitor.scope" . }}
+          - name: SNYK_DEPLOYMENT_NAMESPACE
```

Comment:
what is this for?

Reply:
but it isn't used anywhere?

Reply:
The snyk/config library strips all env vars with the `SNYK_` prefix so only the rest is left, in code we refer to it as `config.DEPLOYMENT_NAMESPACE` instead of `config.SNYK_DEPLOYMENT_NAMESPACE`